src/translator/translate.py

```diff
@@ -50,6 +49,25 @@ def translate_text(
         raise e
 
 
+def should_translate_text(text: str) -> bool:
+    """
+    Identify whether we should translate text.
+
+    For example punctuation and numbers shouldn't be translated as they are the same in
+    most languages.
+    """
+    if all(char in string.punctuation for char in text):
+        return False
+
+    try:
+        float(text)
+        return False
+    except ValueError:
+        pass
+
+    return True
```

Comment:
I think it'd be good to handle any combination of numbers and punctuation here, e.g. `$35 (£25)`. this might be better written with one regex expression?